canonical/lxd-ui
# Confirmation modals: copy updates

## Done

- Updated copy in the confirmation modals to get rid of the "Are you sure?" question.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - Click on the buttons that require a confirmation before executing some action.
    - Check the new copy in the confirmation modal.
